codegen/arrai/grpc_client.arrai

```diff
@@ -0,0 +1,98 @@
+# https://github.com/anz-bank/sysl/blob/master/pkg/sysl/sysl.proto
+# Sysl data structure to handle
+\sysl (
+    grpcclient: \a \name \proj_location
```

Comment:
This is more of a clarification, how are these "\a \name \proj_location" bound to command line params?

Reply:
They're not, they are just parameters into this function call. The caller probably obtains the information from stdargs or stdin though.

Reply:
Ok